aptos-move/framework/aptos-framework/sources/code.move

```diff
@@ -215,8 +215,8 @@ module aptos_framework::code {
 
         let registry = borrow_global_mut<PackageRegistry>(code_object_addr);
         vector::for_each_mut<PackageMetadata>(&mut registry.packages, |pack| {
-            let package: &mut PackageMetadata = pack;
-            package.upgrade_policy = upgrade_policy_immutable();
+            let package_: &mut PackageMetadata = pack;
```

Comment:
This cannot work, we cannot break any user source. `package` must be dealt with using a soft keyword. I thought we discussed this already before?

Reply:
done. Forgot to revert when I removed `package` as a keyword.

Reply:
Also added a test `tests/checking/visibility-checker/parser_package_keyword.move` to make sure we can use "package" as function names, parameter/variable names, and module names.